mobile/native/src/trade/order/handler.rs

```diff
@@ -108,6 +108,7 @@ pub async fn submit_order_internal(
     let url = Url::parse(&url).expect("correct URL");
     let orderbook_client = OrderbookClient::new(url);
 
+    set_order_to_open_and_update_ui(order.id).map_err(SubmitOrderError::Storage)?;
```

Comment:
it probably doesn't hurt, but you do it twice now. Second time is on line 134

Reply:
Argh! 🙈 It absolutely matters! Thanks for spotting it.. this wouldn't have fixed a thing 🤦